addons/l10n_vn/demo/demo_company.xml

```diff
@@ -8,7 +8,7 @@
         <field name="country_id" ref="base.vn"/>
         <field name="state_id" ref="base.state_vn_VN-06"/>
         <field name="zip">76463</field>
-        <field name="phone">+84 91 234 56 78</field>
+        <field name="phone">084 91 234 56 78</field>
```

Comment:
Because the EDI does not support + and it would be better if our demo data work out of the box for it when possible.
Maybe a better solution would be to manage the + in the edi code, not quite sure of that.